demo/demo_args.c

```diff
@@ -1125,6 +1131,36 @@ int demo_parse_argv(cmdline_args_t *parsed_args, int argc, char *argv[]) {
         case 305:
             parsed_args->alternative_logger = true;
             break;
+        case 306: {
+            const size_t identity_length = strlen(optarg);
+            if (parsed_args->connection_args.public_cert_or_psk_identity != NULL
+                    || identity_length == 0) {
+                demo_log(ERROR, "Invalid identity either identity was set "
+                                "twice or empty parameter was passed");
+                goto finish;
+            }
+            clone_buffer(
+                    &parsed_args->connection_args.public_cert_or_psk_identity,
+                    &parsed_args->connection_args
+                             .public_cert_or_psk_identity_size,
+                    optarg, identity_length);
+            break;
+        }
+        case 307: {
+            const size_t key_length = strlen(optarg);
+            if (parsed_args->connection_args.private_cert_or_psk_key != NULL
+                    || key_length == 0) {
+                demo_log(ERROR, "Invalid key, either key was set "
+                                "twice or empty parameter was passed");
+                goto finish;
+            }
+            clone_buffer(
+                    &parsed_args->connection_args.private_cert_or_psk_key,
+                    &parsed_args->connection_args
+                             .private_cert_or_psk_key_size,
```

Comment:
Nitpick: this can fit in 80 columns without line wrap

Reply:
Is there any chance of getting clang-format file ? :D

Reply:
Here's the clang-format file: https://gist.github.com/kFYatek/8b716be6e6fff080a21c97101f7a97d6
However it targets a custom fork of clang-format 6.0 that you can get from https://github.com/kFYatek/clang or https://github.com/dextero/clang/tree/format/dextero... for those changes you're making it's likely that upstream clang-format will work as well, but you'll probably need to remove some of the config options.